autoload/minimap/vim.vim

```diff
@@ -63,6 +63,7 @@ function! s:toggle_window() abort
         return
     endif
 
+    let g:id_list = []
```

Comment:
This currently resides in `s:toggle_window()`; it should be moved to `s:open_window()` so that it always triggers when the minimap is opened, and not only when it is toggled.